Library/ViewModels/PledgeViewModel.swift

```diff
@@ -156,6 +160,10 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
     let pledgeAmountIsValid = self.pledgeAmountDataSignal
       .map { $0.isValid }
 
+    pledgeAmountIsValid.observeValues { v in
```

Comment:
Should we remove this?

Reply:
lol who dis?

Reply:
Used for debug purposes and missed it during cleanup. 🤦‍♂